scipy/interpolate/tests/test_gil.py

```diff
@@ -28,7 +28,7 @@ def run(self):
 
         return WorkerThread()
 
-    @pytest.mark.slow
+    @pytest.mark.xslow
```

Comment:
This one seems fine to mark as `xslow` indeed. Kinda pointless to run really slow tests that are marked as `xfail`.